packages/atoms/src/NumPadButton.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Pressable, PressableProps } from 'react-native';
+
+import { Text } from './Text';
+import { useNativeStyles, NativeStyleObject, prepareNativeStyle } from '@trezor/styles';
+
+export interface NumPadButtonProps extends Omit<PressableProps, 'style'> {
+    title: string;
+    onPress: () => void;
+    variant?: NumPadButtonVariant;
+    style?: NativeStyleObject;
+}
+
+export type NumPadButtonVariant = 'primary' | 'secondary' | 'default';
+
+export type NumPadButtonStyleProps = {
+    variant: NumPadButtonVariant;
+};
+
+const BUTTON_SIZE = 48;
+const BUTTON_FONT_SIZE = 34;
+
+export const numPadButtonStyle = prepareNativeStyle<NumPadButtonStyleProps>(
+    (utils, { variant }) => {
+        const numPadButtonColorSchemeStyles: Record<NumPadButtonVariant, NativeStyleObject> = {
+            primary: {
+                backgroundColor: utils.colors.green,
+            },
+            secondary: {
+                backgroundColor: utils.colors.gray300,
```

Comment:
I wasn't able to find those variants in Figma, could you point me to them? :)

Reply:
Here is the screenshot of the variants that I was implementing. The gray one is for deletion and will probably be used for active state as well, but that I'm not so sure of yet and will need to check later as well.
https://tppr.me/4QY9j

Reply:
That's the "pressed" state :) 
<img width="267" alt="image" src="https://user-images.githubusercontent.com/45338719/168774288-f5a57842-710c-4791-8665-8c05d4c14948.png">

Reply:
5e86703